web/controllers/redcap/helpers.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ResearchResource.Redcap.Helpers do
+  def user_to_record(user) do
+    %{
+      record_id: user.ttrrid,
+      first_name: user.first_name,
+      last_name: user.last_name,
+      email: user.email
+    }
+  end
+
+  # convert yes/no to 1/0
+  # %{"consent_1" => "Yes", "consent_2" => "No", ... } to %{"consent_1" => 1, "consent_2" => 0, ... }
```

Comment:
Maybe these should be in `@doc` tags. See [Writing Documentation](https://hexdocs.pm/elixir/writing-documentation.html)